app/components/show/item/details_component.rb

```diff
@@ -16,6 +16,10 @@ def render?
       delegate :object_type, :created_date, :preservation_size, to: :@solr_document
       delegate :state_service, to: :@presenter
 
+      def manage_catalog_record_id_label
```

Comment:
Oh, good call. Yes, it should be `catalog_record_id_label`. Since only a handful of components use this method, I was not in favor of using inheritance or a mixin.